cmd/run.go

```diff
@@ -76,6 +76,10 @@ func init() {
 	runCmd.PersistentFlags().Uint64("min-satoshis-for-profile", 50000,
 		"Users won't be able to create a profile unless they buy this "+
 			"amount of satoshis or provide a phone number.")
+	// How many times can a phone number be used to receive starter DESO.
+	runCmd.PersistentFlags().Uint64("phone-number-use-threshold", 1, "A phone number will "+
```

Comment:
Let's set the default to 10 so we don't have to configure anything in infra. If it's too much we'll lower it with a flag.